src/server/config.rs

```diff
@@ -183,6 +193,10 @@ impl Default for Config {
             advertise_status_addr: DEFAULT_ADVERTISE_LISTENING_ADDR.to_owned(),
             status_thread_pool_size: 1,
             max_grpc_send_msg_len: DEFAULT_MAX_GRPC_SEND_MSG_LEN,
+            raft_client_grpc_send_msg_buffer: 256 * 1024,
+            raft_client_queue_size: 4096,
+            raft_msg_max_batch_size: 128,
+            raft_msg_flush_delay_us: 500,
```

Comment:
This looks huge

Reply:
It will be removed. See https://github.com/tikv/tikv/pull/10226